### PR TITLE
SorrTask813_Replace_sorrentum_reference_to_kaizenflow

### PR DESCRIPTION
**Pull request by Sameep2808**
#813 

This PR contains all files from [on-boarding](https://github.com/kaizen-ai/kaizenflow/tree/master/docs/onboarding)

### CODE REVIEW

File: docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md [hunk 1]
```diff
@@ -70,20 +70,20 @@
...
+  - [ ] Make sure the [Docker dev container](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/work_tools/all.docker.how_to_guide.md) works
+  - [ ] Make sure the [unit tests](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/coding/all.write_unit_tests.how_to_guide.md) run successfully
+  - [ ] Read [Kaizenflow Python coding style guide](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/coding/all.coding_style.how_to_guide.md)
+  - [ ] Fork, star, watch the Kaizenflow repo so that GitHub promotes our repo (we gotta work the system)
+  - [ ] Learn about the [Morning Email](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/work_organization/all.team_collaboration.how_to_guide.md#morning-email)
+  - [ ] How to do a [review](https://github.com/kaizen-ai/kaizenflow/blob/master/docs/coding/all.submit_code_for_review.how_to_guide.md)
+  - [ ] If you are graduating soon and you would like to get a full-time job in one of the companies in the Kaizenflow ecosystem reach out to GP at [gp@kaizenflow.org](mailto:gp@kaizenflow.org)
```
Comment (Sameep2808) [inline docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md:82]: @gpsaggese can you confirm the new email id

File: docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md [hunk 2]
```diff
@@ -55,7 +55,7 @@
 
 - Upon identifying the top candidate(s), the HiringMeister is expected to notify
   GP for sending invitations to the
-  [sorrenutum](https://github.com/sorrentum/sorrentum) GitHub repository
+  [sorrenutum](https://github.com/kaizen-ai/kaizenflow) GitHub repository
```
Comment (samarth9008) [inline docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md:58]: Missed it

File: docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md [hunk 3]
```diff
@@ -248,20 +248,20 @@ Happy coding!
...
   ```
 
-- Pull the latest Sorrentum image; this is done once
+- Pull the latest Kaizenflow image; this is done once
 
   ```bash
   > i docker_pull
   or
-  > docker pull sorrentum/cmamp:latest
+  > docker pull kaizenflow/cmamp:latest
```
Comment (samarth9008) [inline docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md:256]: This should be reverted as our base image is still sorrentum

Can you add a TODO above it for this change?

File: docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md [hunk 4]
```diff
@@ -248,20 +248,20 @@ Happy coding!
...
   ```
 
 - Pull the latest `dev_tools` image containing the linter; this is done once
 
   ```bash
   > i docker_pull_dev_tools
   or
-  > docker pull sorrentum/dev_tools:prod
+  > docker pull kaizenflow/dev_tools:prod
```
Comment (samarth9008) [inline docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md:264]: same as above

**Review by samarth9008 — COMMENTED** [review]
Few nits 

Are all the links working? Did you check?

File: docs/onboarding/all.development_documents.reference.md [hunk 5]
```diff
@@ -14,21 +14,21 @@
 
 # On-boarding
 
-- [Signing up for Sorrentum](/docs/onboarding/sorrentum.signing_up.how_to_guide.md)
+- [Signing up for Kaizenflow](/docs/onboarding/kaizenflow.signing_up.how_to_guide.md)
```
Comment (gpsaggese) [inline docs/onboarding/all.development_documents.reference.md:17]: Did we renamed these files?

File: docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md [hunk 6]
```diff
@@ -70,20 +70,20 @@
...
-  - [ ] Make sure the [Docker dev container](https://github.com/sorrentum/sorrentum/blob/master/docs/work_tools/all.docker.how_to_guide.md) works
-  - [ ] Make sure the [unit tests](https://github.com/sorrentum/sorrentum/blob/master/docs/coding/all.write_unit_tests.how_to_guide.md) run successfully
-  - [ ] Read [Sorrentum Python coding style guide](https://github.com/sorrentum/sorrentum/blob/master/docs/coding/all.coding_style.how_to_guide.md)
-  - [ ] Fork, star, watch the Sorrentum repo so that GitHub promotes our repo (we gotta work the system)
-  - [ ] Learn about the [Morning Email](https://github.com/sorrentum/sorrentum/blob/master/docs/work_organization/all.team_collaboration.how_to_guide.md#morning-email)
-  - [ ] How to do a [review](https://github.com/sorrentum/sorrentum/blob/master/docs/coding/all.submit_code_for_review.how_to_guide.md)
-  - [ ] If you are graduating soon and you would like to get a full-time job in one of the companies in the Sorrentum ecosystem reach out to GP at [gp@sorrentum.org](mailto:gp@sorrentum.org)
+  - [ ] Acknowledge the pledge to put time in the project [here](https://github.com/kaizen-ai/kaizenflow/blob/master/README.md#Important)
```
Comment (gpsaggese) [inline docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md:73]: same here

File: docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md [hunk 7]
```diff
@@ -22,11 +22,11 @@
 
 <!-- tocstop -->
 
-# Set up Sorrentum development environment
+# Set up Kaizenflow development environment
```
Comment (gpsaggese) [inline docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md:25]: -> KaizenFlow everywhere when in text

File: docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md [hunk 8]
```diff
@@ -261,6 +262,7 @@ Happy coding!
   ```bash
   > i docker_pull_dev_tools
   or
+  # TODO(Sameep): Update to kaizenflow once docker is updated.
```
Comment (gpsaggese) [inline docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md:265]: You can file a bug to rename/move the Docker hub

**Review by gpsaggese — COMMENTED** [review]
Going in the right direction, but let's review the PR with fresh eyes to catch the imperfections

We need 100% attention to details

File: docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md (hunk 7 above)
Comment (samarth9008) [reply docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md:25]: @Sameep2808 Can you change this? Make sure to not change it in the repo links.

File: docs/onboarding/all.development_documents.reference.md (hunk 5 above)
Comment (samarth9008) [reply docs/onboarding/all.development_documents.reference.md:17]: Yes

File: docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md (hunk 7 above)
Comment (samarth9008) [reply docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md:25]: Filenames should be kaizenflow

**Review by samarth9008 — COMMENTED** [review]
Just comments from GP

File: docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md (hunk 6 above)
Comment (Sameep2808) [reply docs/onboarding/kaizenflow.hiring_meister.how_to_guide.md:73]: Yes did update and all the links are working

File: docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md (hunk 8 above)
Comment (Sameep2808) [reply docs/onboarding/kaizenflow.set_up_development_environment.how_to_guide.md:265]: https://github.com/kaizen-ai/kaizenflow/issues/836

**Review by samarth9008 — COMMENTED** [review]
LG merging

**Pull request merged**